test/e2e/README.md

```diff
@@ -187,6 +187,12 @@ manifest to the control-plane Docker container:
 go test -v antrea.io/antrea/test/e2e -provider=kind
```

Comment:
shouldn't we just add a timeout value here?

Reply:
yes, we can do that, but my original purpose is to remind the user there is also a timeout setting impact the results, so they can notice this even they are running some separate test cases. for example, the Networkpolicy related take over 15mins

Reply:
let me add the timeout here and refine the follow descriptions.